### DISCUSSION TITLE
Unable build the project

### DISCUSSION THREAD

**Discussion by sivakothuri** [eclipse-emfcloud/emfcloud#169] [ModelServer] [answered]
While building the model server project using maven command  **mvn -U clean install**
getting the following error
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.1:check (default) on project org.eclipse.emfcloud.modelserver.example: Failed during checkstyle execution: Failed during checkstyle configuration: unable to parse configuration stream: No subject alternative DNS name matching checkstyle.org found. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :org.eclipse.emfcloud.modelserver.example`
Java: 11
Maven: 3.8.5

How to resolve it?

**Comment by sgraband** [comment, marked as answer]
Hey @sivakothuri,

Could you try to revert to Maven version 3.8.4? See [here](https://github.com/eclipse-emfcloud/emfcloud-modelserver/issues/203) for more information.

Hope this helps!
